src/annotator/integrations/html.js

```diff
@@ -117,9 +117,23 @@ export class HTMLIntegration {
     const padding = 12;
     const rightMargin = sidebarWidth + padding;
 
+    /**
+     * @param {HTMLElement} element
+     */
+    const computeLeftMargin = element =>
```

Comment:
For internal functions that only have one parameter, I suggest to put the whole JSDoc comment on one line to save space (`/** @param {HTMLElement} element */`).